src/components/table-hoc/TableHOC.tsx

```diff
@@ -0,0 +1,49 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import {ReduxConnect} from '../../utils/ReduxUtils';
+import {ActionBarConnected} from '../actions/ActionBarConnected';
+
+export interface ITableHOCOwnProps {
+    id: string;
+    hasActionButtons?: boolean;

```

Comment:
why do you need to have this prop? can't you use `_.isEmpty(this.props.actions)` ?

Reply:
I think i know, I'd suggest you leave actions as undefined and not `[]` and do a falsy check with `_.isArray` instead of having another prop `hasActionButtons` , if we pass an empty array the action bar will still show

Reply:
For the moment I don't need it, but I was thinking about when I'll connect the rows with "selectable" how I'll be able to add the rows actions. I can probably delete it in this PR and add it when needed

Reply:
in that sense, maybe we should have a `getActions` method instead of `actions`, not every row will have the same actions

Reply:
Removed for now